Rakefile

```diff
@@ -19,7 +21,7 @@ rescue LoadError
 end
 
 desc 'Run tests in generated test Rails app with generated Solr instance running'
-task ci: [:environment] do
+task ci: [:rubocop, :environment] do
```

Comment:
OK. I assume it's because you prefer to defer Rubocop-related changes until you're sure everything is :green_apple: and :smiley:?